consumer/tests/src/test_twitter_consumer.e

```diff
@@ -123,14 +123,14 @@ feature {NONE} -- Initialization
 				across
 					ctx.query_parameters as q
 				loop
-					tb.force (q.item, q.key)
+--					tb.force (q.item, q.key)
```

Comment:
is this commented line expected?

Reply:
not sure, it seems obsolete code, I will check.